tests/specs/types/ConditionalType/ConditionalType_LinePerExpression_True.txt

```diff
@@ -0,0 +1,20 @@
+~~ lineWidth: 80, conditionalType.linePerExpression: true ~~
+== should use a line per expression when multiLine ==
+type Test = A extends B
+    ? C extends D
+    ? c
+    : testingTesting
+    : testingTestingTestingTesting;
+
+[expect]
+type Test = A extends B
+    ? C extends D
+    ? c
+    : testingTesting
+    : testingTestingTestingTesting;
```

Comment:
@declanvong sorry for my delay here. I just added this file and test, but it fails. I think this is the expected output?

Reply:
Yeah, looks like that maybe one of the indent conditions is listening to linePerExpression instead of useNestedIndentation? I'll take a look

Reply:
Fixed, but important to note that this change is not backwards compatible, it changes existing formatting, so it'd be a version bump. 
There's a test with some existing formatting that I've updated (`ConditionalType_All.txt::should format an individual condition that goes over the limit`).

Since both `linePerExpression` and `useNestedIndentation` are not enabled in that test, it goes from being formatted originally like this:
```
type Type<T> = T extends string ? number
    : T extends hereIsAVeryLongExtendsClause
        ? testingThis
    : boolean;
```

to this (flattened indentation):
```
type Type<T> = T extends string ? number
    : T extends hereIsAVeryLongExtendsClause
    ? testingThis
    : boolean;
```

Enabling `useNestedIndentation` _without_ `linePerExpression` gives:
```
type Type<T> = T extends string ? number
    : T extends hereIsAVeryLongExtendsClause
        ? testingThis
        : boolean;
```
which also seems correct.

So there's no way to get the original formatting in this new world, but maybe that's good? if the original formatting was always not technically correct or wanted. after all, that trailing `: boolean` really is part of the inner conditional, not the outer one, so I feel like the new formatting is better anyway as it makes that more obvious.